src/metabase/api/getting_started.clj

```diff
@@ -5,16 +5,17 @@
             [metabase.models
              [interface :as mi]
              [setting :refer [defsetting]]]
+            [puppetlabs.i18n.core :refer [tru]]
             [toucan.db :as db]))
 
 (defsetting getting-started-things-to-know
-  "'Some things to know' text field for the Getting Started guide.")
+  (tru "'Some things to know' text field for the Getting Started guide."))
```

Comment:
`'` is a special character in gettext, so it needs to be escaped with a `'`. The above should be `''Some things to know''`